portfolio/src/main/webapp/css/global.css

```diff
@@ -0,0 +1,193 @@
+html, body {
+  font-family: 'Raleway', sans-serif;
+  margin: 0px;
+
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+}
+
+#wrapper {
+  flex: 1;
+
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: auto;
+}
+
+.divider {
+  width: 280px;
+  display: flex;
+  justify-content: center;
+  margin: auto;
+}
+
+.footer {
+  font-family: 'Raleway';
+  font-size: 12px;
+
+  height: 65px;
+  width: 300px;
+  align-items: center;
+
+  margin: auto;
+  padding-left: 6%;
+  padding-right: 6%;
+  /* padding-top: 1%; */
+  margin-bottom: 1%;
+}
+
+.links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.colored {
+  color: #99c4bb;
+  font-weight: bold;
+}
+
+a {
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+img {
+  border-radius: 10px;
+}
+
+a {
+	color:#8082D7;
+	transition:color linear 0.175s;
+}
+
+a:hover, .menu .current-item a {
+	text-decoration:none;
+	color:#89BBE7;
+}
+
+.footer a {
+  color: black;
+  transition:color linear 0.175s;
+}
+
+.footer a:hover {
+  color:#89BBE7;
+}
+
+@media screen and (max-width: 320px) {
+	.footer {
```

Comment:
indentation appears to be a bit off starting around here. Could you run a code formatter across the project? prettier.io is a good one to start with.

Reply:
Sure I'll give that a go